playbarComponent/client/src/components/App.jsx

```diff
@@ -51,118 +51,122 @@ class App extends Component {
-    //         currentSong: songsList[0],
-    //         currentSongTitle: songNamesList[0],
-    //         currentArtist: artistList[0],
-    //         currentArt: artList[0],
-    //         artList: artList,
-    //         artistList: artistList,
-    //         songNamesList: songNamesList,
-    //         currentTime: time,
-    //         duration: songsList[0].duration
-    //       },
-    //       () => {
-    //         console.log('current song', this.state.currentSong);
-    //         let d = this.state.currentSong.duration;
-    //         console.log('d: ', d);
-    //         let dur = this.fancyTimeFormat(this.state.currentSong.duration);
-    //         console.log('duration: ', dur);
-    //       }
-    //     );
-    //   })
-    //   .catch(err => {
-    //     console.log('Could not load songs from db!');
-    //   });
-
-    list = [
-      {
-        lengthString: 'Please choose a song first!',
-        currentTime: 0,
-        name: 'All I Got',
-        songNameURL:
-          'https://5cloudaudio.s3-us-west-1.amazonaws.com/All_I_Got.mp3',
-        artist: 'Said The Sky',
-        songArtURL:
-          'https://i1.sndcdn.com/artworks-87d2dfb3-404e-46c4-a9c1-ca749b012f52-0-t50x50.jpg',
-        duration: 366
-      },
-      {
-        lengthString: 'Please choose a song first!',
-        currentTime: 0,
-        name: 'Flicker',
-        songNameURL:
-          'https://5cloudaudio.s3-us-west-1.amazonaws.com/flicker.mp3',
-        artist: 'Porter Robinson',
-        songArtURL:
-          'https://i1.sndcdn.com/artworks-000376950786-x9c78f-t50x50.jpg',
-        duration: 316
-      },
-      {
-        lengthString: 'Please choose a song first!',
-        currentTime: 0,
-        name: 'Say My Name',
-        songNameURL:
-          'https://5cloudaudio.s3-us-west-1.amazonaws.com/Say_My_Name.mp3',
-        artist: 'Odesza',
-        songArtURL: 'https://i1.sndcdn.com/artworks-ILVTUNh2LAia-0-t50x50.jpg',
-        duration: 266
-      }
-    ];
-
-    let songsList = [];
-    let artList = [];
-    let artistList = [];
-    let songNamesList = [];
-    let duration = [];
-    for (let i = 0; i < list.length; i++) {
-      let song = new Audio(list[i].songNameURL);
-      // console.log('song: ', song);
-      songsList.push(song);
-      artList.push(list[i].songArtURL);
-      artistList.push(list[i].artistName);
-      songNamesList.push(list[i].songName);
-      duration.push(list[i].duration);
-    }
-
-    var time = this.fancyTimeFormat(0);
-    // console.log('songsList: ', songsList);
-    this.setState({
-      currentList: songsList,
-      currentSong: songsList[0],
-      currentArt: artList[0],
-      currentArtist: list[0].artist,
-      currentSongTitle: list[0].name,
-      artistList: artistList,
-      songNamesList: songNamesList,
-      currentTime: time,
-      duration: duration[0]
-    });
+    axios
+
+      .get('http://localhost:5002/query/getSong/song_0001')
```

Comment:
I noticed you used dynamic url handling in your express server but hard-code in the song_id here. Do you want to use the dynamic functionality of the url?